lib/routes/cjlu/yjsy/index.ts

```diff
@@ -50,17 +82,33 @@ export const route: Route = {
 
 async function handler(ctx) {
     const cate = ctx.req.param('cate');
+    const limit = ctx.req.query('limit') ? Number.parseInt(ctx.req.query('limit'), 10) : 10;
+    const url = `${host}index/${cate}.htm`;
 
-    const response = await ofetch(`${cate}.htm`, {
-        baseURL: `${host}/index/`,
-        responseType: 'text',
+    const { page, destory, browser } = await getPuppeteerPage(url, {
+        onBeforeLoad: async (page) => {
+            await page.setExtraHTTPHeaders(headers);
+            await page.setUserAgent(headers['User-Agent']);
+            await page.setRequestInterception(true);
+            page.on('request', (request) => {
+                excludeResourceTypes.has(request.resourceType()) ? request.abort() : request.continue();
+            });
```

Comment:
Explicitly list which [request resource type](https://chromedevtools.github.io/devtools-protocol/tot/Network/#type-ResourceType) is allowed instead of blocking specific resource type